cl/corpus_importer/management/commands/recap_into_opinions.py

```diff
@@ -1,5 +1,9 @@
+from typing import Optional
```

Comment:
In Python 3.10 and later, you can express optional types more concisely using union syntax (`type | None`) instead of relying on the `typing.Optional `import.

For example, instead of:

```python
from typing import Optional

def function_1(value: Optional[int]) -> str:
    # ...
```

You can write:
```python
def get_value(value: int | None) -> str:
  # ...
```

Reply:
yes thank you - I thought I included that from your branch